README.md

```diff
@@ -12,8 +12,6 @@ with an express/sinatra style syntax for defining routes and their handlers.
 Pretender will temporarily replace native XMLHttpRequest and Fetch , intercept
 all requests, and direct them to little pretend service you've defined.
 
-**:warning: Pretender only works in the browser!**
-
```

Comment:
Removing this would seem to indicate that pretender works in non-browser environments now. Is this the case?

Reply:
Nope, I had the wrong idea about this when I started to play with the Rollup config. Reverting, thanks!